packages/flutter_tools/lib/src/cache.dart

```diff
@@ -83,11 +83,25 @@ class Cache {
       _artifacts.add(LinuxEngineArtifacts(this));
       _artifacts.add(LinuxFuchsiaSDKArtifacts(this));
       _artifacts.add(MacOSFuchsiaSDKArtifacts(this));
+      for (String artifactName in iosUsbArtifactNames) {
+        _artifacts.add(IosUsbArtifacts(artifactName, this));
+      }
     } else {
       _artifacts.addAll(artifacts);
     }
   }
 
+  /// The names of iOS/USB utility artifacts in the cache.
+  static const List<String> iosUsbArtifactNames = <String>[
```

Comment:
Can this be a static const of `IosUsbArtifacts`?

Reply:
done